docs/dev/analysis/features/text/pictures-in-runs.rst

```diff
@@ -0,0 +1,105 @@
+
+
+Word Behavior
+-------------
+
+In theory, a Run can contain multiple Drawings.  In practice, I have not found a way to create a Run with more than a single Drawing.
+
+
+Protocol
+--------
+
+.. highlight:: python
+
+To stay parallel with document.add_picture() behavior, which creates an Inline object, the Run should have an Inlines property, producing a list of Inline objects.
+
+  >>> inlines = run.inline_lst
+  >>> inline = inlines[0]
+  >>> inline
+  <CT_Inline '<wp:inline>' at 0x43de510>
+  
+From here, we can access properties which characterize the image.
+
+  >>> rId = inline.graphic.graphicData.pic.blipFill.blip.embed
+  >>> rId
+  'rId7'
+  >>> cx, cy = inline.graphic.graphicData.pic.spPr.cx, inline.graphic.graphicData.pic.spPr.cy
+  >>> cx.inches, cy.inches
+  (4.61111111111, 2.33333333333)
+  >>> document.part.rels[rId]._target.image.content_type
+  'image/png'
+  >>> rels[rId]._target.image.filename, rels[rId]._target.image.ext
+  ('image.png', 'png')
+  >>> originalx, originaly = rels[rId]._target.image.width, rels[rId]._target.image.height
+  >>> originalx, originaly
+  (4.61111111111, 2.33333333333)
+  >>> image_data = rels[rId]._target.image.blob
+  >>> len(image_data)
+  23442
+  
+This interface, while (mostly) following Microsoft's object model (except for leaving out the Drawing level), seems unfriendly to the python_docx user.  Therefore, I would recommend the following additional interface elements.
+
+  >>> pictures = run.picture_lst
+  >>> picture = pictures[0]
+  >>> picture.rId
+  'rId7'
+  >>> picture.cx.inches, picture.cy.inches, picture.originalx.inches, picture.originaly.inches
+  (4.61111111111, 2.33333333333, 4.61111111111, 2.33333333333)
+  >>> picture.content_type
+  'image/png'
+  >>> picture.filename, picture.ext
+  ('media/image.png', 'png')
+  >>> image_data = picture.image_data
+  >>> f = open('image_export.%s' % picture.ext, 'wb')
+  >>> f.write(image_data)
+  >>> f.close()
+  (produces a PNG image file named 'image_export.png' containing the image) 
+
+
+Minimal XML
+-----------
+
+.. highlight:: xml
+
+<w:drawing>
+  <wp:inline>
+    <a:graphic>
+      <a:graphicData>
+        <pic:pic>
+          <pic:blipFill>
+            <a:blip r:embed='rId7'>
+            </a:blip>
+          </pic:blipFill>
+          <pic:spPr>
+            (information about image offset and size)
+          </pic:spPr>
+        </pic:pic>
+      </a:graphicData>
+    </a:graphic>
+  </wp:inline>
+</w:drawing>
+
+See documentation of CT_Inline, CT_GraphicalObject, CT_GraphicalObjectData, CT_Picture, CT_BlipFillProperties, and CT_Blip for more information.
+                   
+                   
+Resources
+---------
```

Comment:
The XML Schema excerpt is an absolutely mandatory section in an analysis document. You can't get the oxml right without it. The Inline shape page this content will be added to already has some excerpts. You'll want to make sure it has everything you're going to need.